Models/Certificate.swift

```diff
@@ -142,31 +142,43 @@ private enum MethodsForV1_1 {
     }
 
     static func parse(recipientJSON: AnyObject?) -> Recipient? {
+
         guard let recipientData = recipientJSON as? [String : AnyObject],
             let identityType = recipientData["type"] as? String,
             let familyName = recipientData["familyName"] as? String,
             let givenName = recipientData["givenName"] as? String,
-            let isHashed = recipientData["hashed"] as? String,
+            let isHashedObj = recipientData["hashed"] as AnyObject?,
```

Comment:
This conversion is unnecessary. `recipientData` says the values will be of type `AnyObject`. So `recipientData["hashed"]` will already be of type `AnyObject` and won't need the `as` cast.